doc/src/manual/modules.md

```diff
@@ -357,26 +357,26 @@ Other known potential failure scenarios include:
 Several additional restrictions are placed on the operations that can be done while precompiling
 code to help the user avoid other wrong-behavior situations:
 
-1. Calling `eval` to cause a side-effect in another module. This will also cause a warning to be
+1. Calling [`eval`](@ref) to cause a side-effect in another module. This will also cause a warning to be
    emitted when the incremental precompile flag is set.
 2. `global const` statements from local scope after `__init__()` has been started (see issue #12010
    for plans to add an error for this)
-3. Replacing a module (or calling `workspace()`) is a runtime error while doing an incremental precompile.
+3. Replacing a module (or calling [`workspace()`](@ref)) is a runtime error while doing an incremental precompile.
 
 A few other points to be aware of:
 
 1. No code reload / cache invalidation is performed after changes are made to the source files themselves,
-   (including by `Pkg.update`), and no cleanup is done after `Pkg.rm`
+   (including by [`Pkg.update`](@ref)), and no cleanup is done after [`Pkg.rm`](@ref)
 2. The memory sharing behavior of a reshaped array is disregarded by precompilation (each view gets
    its own copy)
-3. Expecting the filesystem to be unchanged between compile-time and runtime e.g. `@__FILE__`/`source_path()`
+3. Expecting the filesystem to be unchanged between compile-time and runtime e.g. [`@__FILE__`](@ref)/[`source_path()`](@ref)
```

Comment:
`source_path` isn't exported (see https://travis-ci.org/JuliaLang/julia/jobs/186485491#L771) so will need to be written as `` [`Base.source_path`](@ref) `` to be visible to Documenter.